### ISSUE TITLE
[ED][Captions]

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-media-intro#103]
> Note: If a video has no narration or dialog, inform viewers...

Consider putting this under a heading so it doesn't get missed.

**Comment by kakinney** [comment]
I agree with making that Note a heading but you are going to have to rethink the heading structure on that page to accommodate it.  There is the H1 for the page and no other heading levels before you get to that note.
